server/index.js

```diff
@@ -1,16 +1,18 @@
 const express = require('express');
+require('dotenv').config();
```

Comment:
You able to put this at the very top? Kind of feels out of place, just an opinion.

Reply:
Recommended usage of this package is 'As early as possible in your application, require and configure dotenv.' I suppose to allow for variables to change when deployed.

Reply:
https://www.npmjs.com/package/dotenv

Reply:
Does the code break when requiring dotenv before initializing express? Since express isn't quite invoking anything yet.